README.md

```diff
@@ -64,26 +66,29 @@ Whenever you change the `name` property - or attribute - the component will re-r
   - [Examples](#examples)
     - [Counter](#counter)
   - [API](#api)
-    - [`define(name, definition)`](#definename-definition)
-      - [`prototype`](#prototype)
-      - [`props`](#props)
-        - [`attribute`](#attribute)
-        - [`coerce`](#coerce)
-        - [`default`](#default)
-        - [`deserialize`](#deserialize)
-        - [`get`](#get)
-        - [`initial`](#initial)
-        - [`serialize`](#serialize)
-        - [`set`](#set)
-      - [`created`](#created)
-      - [`updated`](#updated)
+    - [Using the Platform](#using-the-platform)
+    - [`constructor` - supercededs `static created()`](#constructor---supercededs-static-created)
```

Comment:
`supercededs` is a really weird word. My English is not as good as I hoped or it really just means `supersedes`?